vignettes/adrs.Rmd

```diff
@@ -55,6 +55,7 @@ For example purpose, the SDTM and ADaM datasets (based on CDISC Pilot test data)
 included in `{admiral.test}`---are used.
 
 ```{r message=FALSE}
+library(admiraldev)
```

Comment:
We do not need this unless we call an admiraldev function in the script.

Reply:
i think we need this for `dataset_vignette` right?

Reply:
Yes, you are right. However, I would put it in a separate
````
```{r echo=FALSE}
```
````
block such that it is not visible for the user. We need it for building the vignette but the user does not need it.

Reply:
yes makes sense will do, i just saw that's what Ben did for admiral core